test/end-to-end/core.sh

```diff
@@ -184,6 +184,11 @@ os::cmd::expect_success_and_text "cat '${LOG_DIR}/kubectl-with-token.log'" 'Usin
 os::cmd::expect_success_and_text "cat '${LOG_DIR}/kubectl-with-token.log'" 'kubectl-with-token'
 
 echo "[INFO] Testing deployment logs and failing pre and mid hooks ..."
+# test hook selectors
+os::cmd::expect_success 'oc create -f test/fixtures/complete-dc-hooks.yaml'
```

Comment:
The path to the fixture file should be written relative to `${OS_ROOT}` for stability.